spec/lib/index.spec.js

```diff
@@ -21,15 +21,74 @@ describe("lib/index", () => {
 
       // Invalid options.entry
       expect(() => { new Interlock(_.merge({}, minimalValidConfig, { entry: true })); })
-        .to.throw(Error);
+        .to.throw(Error, "child \"entry\" fails because [\"entry\" must be an object]");
```

Comment:
Meh, this is fine by me.  Catching specific errors is always kind of strange, unless you have an error hierarchy defined.